macOS/geotoad.rb

```diff
@@ -0,0 +1,21 @@
+class Geotoad < Formula
+  desc "Query tool to query the geocaching.com website"
+  homepage "https://github.com/steve8x8/geotoad"
+  url "https://github.com/steve8x8/geotoad/archive/3.25.4.tar.gz"
```

Comment:
You’re right. The correct approach is to create a [tap](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/brew-tap.md), meaning a new repo (`steve8x8/homebrew-geotoad`) and inside put a this file.

Since I cannot PR a repo creation, I submitted it like this, but forgot about that problem that makes it impossible with the way you package (you only release the source and don’t do an actual release wit notes and the download file).

Are yo interested in creating said repo (so I know how to edit this)?